src/components/BreadcrumbsItem/BreadcrumbsItem.tsx

```diff
@@ -26,22 +39,94 @@ export interface Props {
  * import {BreadcrumbsItem} from "@chanzuckerberg/eds";
  * ```
  *
- * A single breadcrumb, to be used in the Breadcrumbs component.
+ * A single breadcrumb subcomponent, to be used in the Breadcrumbs component.
  */
-export const BreadcrumbsItem = ({ className, text, href, ...other }: Props) => {
-  const componentClassName = clsx(styles['breadcrumbs__item'], className, {});
+export const BreadcrumbsItem = ({
+  dropdownMenuItems,
+  className,
+  href,
+  text,
+  variant,
+  ...other
+}: Props) => {
+  const [isActive, setIsActive] = React.useState(false);
+
+  /**
+   * Closes the dropdown menu if there is focus or click outside.
+   */
+  const handleBlur = (event: React.FocusEvent<HTMLElement, Element>) => {
+    if (!event.currentTarget.contains(event.relatedTarget) && isActive) {
+      setIsActive(false);
+    }
+  };
+
+  const componentClassName = clsx(
+    styles['breadcrumbs__item'],
+    variant === 'back' && styles['breadcrumbs__item-back'],
+    className,
+  );
+
+  const ellipsisButtonClassName = clsx(
+    styles['breadcrumbs__link'],
+    styles['breadcrumbs__ellipsis'],
+  );
+
+  const dropdownMenuClassName = clsx(
+    styles['breadcrumbs__dropdown-menu'],
+    isActive && styles['breadcrumbs__dropdown-menu--active'],
+  );
+
+  const getInteractionElement = () => {
+    if (variant === 'collapsed') {
+      /* The collapsed variant is a button with ellipsis. Interaction spawns a dropdown containing the collapsed breadcrumb links. */
+      return (
+        <>
+          <button
+            className={ellipsisButtonClassName}
+            onClick={() => {
+              setIsActive(!isActive);
+            }}
+          >
+            ...
```

Comment:
You're going to need an aria label on this. I just tested it out with voiceover and it just says "button" because a period is just a pause. It also doesn't have the `aria-expanded` info so a user knows it opens a dropdown. Should we be using the `ButtonDropdown` component here instead of using the `DropdownMenu` directly?

Reply:
Yep, will use `ButtonDropdown`. Initially thought that it wouldn't work, but just tried it and worked. Also will add `aria-label="Show more breadcrumbs"` (same as the [atlassian design system](https://atlassian.design/components/breadcrumbs/examples#long-breadcrumbs) lol)